### ISSUE TITLE
Gem Usage

### ISSUE THREAD

**Issue by binyamindavid** [mascott/activestorage-google-drive#1]
How do you use this gem?

**Comment by andreibondarev** [comment]
@mascott Any plans to pick the development of this gem back up?

**Comment by mascott** [comment]
I actually ran into some issues with the way the Google Drive api works compared to what ActiveStorage expects. There's probably a path forward, but I ended up moving to Google Cloud instead.

**Comment by andreibondarev** [comment]
@mascott I see. I was looking to upload files to Google Drive and have them automatically converted to Google Docs equivalents. Wanted to avoid the intermediary Google Cloud step.

**Comment by mascott** [comment]
Yeah, that makes sense. It's been a while since I worked on this gem so I'm trying to remember the specific issue I was running into. If I remember correctly ActiveStorage expects you to use the identifier it provides when saving the file to whatever service you're using, but Google Drive doesn't give you the ability (or at least I couldn't figure out how) to specify the id and instead just returns it's own generated key. I thought about adding my own model to store both, but then I would need to handle that middle step when looking up the file in Google Drive. 

It definitely seemed possible, but wasn't worth the effort when Google Cloud already provided the functionality I needed for the project I was working on.
